client/src/App.js

```diff
@@ -3,15 +3,17 @@ import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
 
 import "./App.css";
+import AssignRoles from './pages/AssignRoles';
+// import Chat from './pages/Chat';
```

Comment:
If this commented out code is not needed, you can remove it, ditto for the below

Reply:
Removed the commented code.